### ISSUE TITLE
draw_geometries not working for simple example

### ISSUE THREAD

**Issue by jungerm2** [intel-isl/Open3D#1987]
**Describe the bug**
Unable to use `draw_geometries(...)`. In fact, the documentation examples fail.

**To Reproduce**
Run this example: http://www.open3d.org/docs/release/tutorial/Basic/visualization.html#Draw-line-set 

**Expected behavior**
A 3D cube should be drawn. Instead this error gets thrown:
```
TypeError                                 Traceback (most recent call last)
<ipython-input-5-72398a1fcf68> in <module>
     30 )
     31 line_set.colors = o3d.utility.Vector3dVector(colors)
---> 32 o3d.visualization.draw_geometries([line_set], zoom=0.8)

TypeError: draw_geometries(): incompatible function arguments. The following argument types are supported:
    1. (geometry_list: List[open3d.open3d.geometry.Geometry], window_name: str = 'Open3D', width: int = 1920, height: int = 1080, left: int = 50, top: int = 50) -> None

Invoked with: [geometry::LineSet with 12 lines.]; kwargs: zoom=0.8
```

**Environment (please complete the following information):**

- Operating system: Ubuntu 18.04
- Python version: Python 3.7
- Open3D version: 0.10.0
- Is this remote workstation?: yes
- How did you install Open3D?: pip

**Comment by alainchau** [comment]
Looks like `draw_geometries` doesn't support the zoom argument anymore and documentation hasn't been updated yet. In the meantime, if you want to alter zoom, you can replace the line 
```o3d.visualization.draw_geometries([line_set], zoom=0.8)```

with
```
vis = o3d.visualization.Visualizer()
vis.create_window()
vis.add_geometry(line_set)
o3d.visualization.ViewControl.set_zoom(vis.get_view_control(), 0.8)
vis.run()
```

**Comment by griegler** [comment]
Same as in #1940. The issue is addressed in #1997.

**Comment by nbalive08** [comment]
> Looks like `draw_geometries` doesn't support the zoom argument anymore and documentation hasn't been updated yet. In the meantime, if you want to alter zoom, you can replace the line
> `o3d.visualization.draw_geometries([line_set], zoom=0.8)`
> 
> with
> 
> ```
> vis = o3d.visualization.Visualizer()
> vis.create_window()
> vis.add_geometry(line_set)
> o3d.visualization.ViewControl.set_zoom(vis.get_view_control(), 0.8)
> vis.run()
> ```

Thanks!!!!!!!
